scripts/foundation-wallet-deployment/config.ts

```diff
@@ -0,0 +1,46 @@
+interface FoundationWalletDeploymentConfig {
+  beneficiary: string;
+  startTimestamp: number;
+  durationSeconds: number;
+  l2ArbitrumGovernor: string;
+  l2GovProxyAdmin: string;
```

Comment:
can we get this in runtime from `l2ArbitrumGovernor` instead of also providing it?

Reply:
importing from deployments json now